docs/Tutorial_Matrix_Profiles_For_Streaming_Data.ipynb

```diff
@@ -191,6 +191,25 @@
     "I_full = mp[:, 1]"
```

Comment:
<p>A minor typo in the inline comment provided in the code cell.&nbsp;</p>

<pre class="ql-syntax" spellcheck="false">Replace matric with matrix
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/972/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>